02-spatial-data.Rmd

```diff
@@ -611,14 +622,12 @@ Simple features are, in essence, data frames with a spatial extension.
 
 ### An introduction to raster
 
-Raster objects in R are supported by the **raster** package. 
-It provides an extensive set of functions to create, read, processed and write of raster datasets.
-Beside the general raster data manipulation, **raster** provides many low level functions that can be used to create and develop new concepts. **raster** also supports work on large raster datasets that are stored on a hard drive, but are too large to fit into memory. 
-Instead of recreating the whole file in RAM, this package extracts information about the structure of the dataset, such as a number of rows and columns, spatial extent and the name of the file.
-When manipulating this dataset, values are read and processed in a small chunk and written either to a specified file on a disk or temporary file.
-
-The **raster** package provides three main classes of objects - `RasterLayer`, `RasterBrick` and `RasterStack`. 
```

Comment:
Agree - duplication of content is a problem in the book we need to keep on top of. Fresh eyes are very much appreciated in this area and I agree these sentences should be deleted. Good change.